src/pages/ProcessDetail.tsx

```diff
@@ -66,31 +67,29 @@ function ProcessDetail({ match, query, setQuery }: IProps) {
     const [observer, setObserver] = useState<IntersectionObserver | undefined>(undefined);
     const [autoScrollToLast, setAutoScrollToLast] = useStorageState(localStorage, "process-autoscroll", true);
 
-    useQuery(["process"], () => apiClient.process(match.params.id), {
-        onSuccess: (processInstance) => {
-            let enrichedProcess = processInstance as ProcessWithDetails;
-            const localStepUserInput: InputForm | undefined = enrichedProcess.form;
-            const localTabs = localStepUserInput ? tabs : ["process"];
-            const localSelectedTab = localStepUserInput ? "user_input" : "process";
-
-            setProductName(productNameById(enrichedProcess.product, products));
-            setCustomerName(organisationNameByUuid(enrichedProcess.customer, organisations));
-            setProcess(enrichedProcess);
-            setStepUserInput(localStepUserInput);
-            setTabs(localTabs);
-            setSelectedTab(localSelectedTab);
-        },
-        onError: (err: any) => {
-            if (err.response.status === 404) {
-                setNotFound(true);
-                setLoaded(true);
-            } else {
-                throw err;
-            }
-        },
+    const processQuery = useQuery(["process"], () => apiClient.process(match.params.id), {
```

Comment:
the original already had this: the cache key is not ok like this:
`["process", match.params.id]` would generate a unique hash!

Reply:
Didn't think about that when updating it yea.

```suggestion
    const processQuery = useQuery(["process", match.params.id], () => apiClient.process(match.params.id), {
```